dbt_project.yml

```diff
@@ -0,0 +1,27 @@
+name: 'snowplow_media_player'
+version: '1.0.0'
+config-version: 2
+require-dbt-version: ">=1.0.0"
+
+profile: 'default'
+
+model-paths: ["models"]
+analysis-paths: ["analyses"]
+test-paths: ["tests"]
+seed-paths: ["seeds"]
+macro-paths: ["macros"]
+snapshot-paths: ["snapshots"]
+
+target-path: "target"
+clean-targets:
+  - "target"
+  - "dbt_packages"
+
+models:
+  +bind: false
+  +materialized: view
+
+vars:
+  snowplow__percent_progress_boundaries: {15: 0.15, 25: 0.1, 50: 0.25, 75: 0.25, 100: 0.25}
```

Comment:
It would be nice to derive the weights from the boundaries meaning the user would only need to supply a list of boundaries

Reply:
In the next commit you will see that the user will only have to fill out a list instead of a dictionary. The weights are automatically calculated within `pivot_base`. One thing to note, assuming that the users do not track `100%` as there is already a separate `ended` event and it is changed to `100%` within the model, I added an if condition to add 100 to it, in case they do not specify it so that the list will be complete. I don't suppose they would not want to see the full picture with the same format. The other option is to not modify the ended events or anything here but keep a separate `ended` field which might be simpler but less coherent for downstream models)